internal/effect/shader.go

```diff
@@ -66,8 +66,24 @@ func convertHSV2RGB(hsv vec3) vec3 {
 }
 
 func Fragment(position vec4, texCoord vec2, color vec4) vec4 {
-	var txtcolor vec4 = imageSrc0At(texCoord)
-	if txtcolor.a < 0.001 {
+	var txtcolor vec4
```

Comment:
if Color == 0 && Brightness == 0, this function should return immediately.

Reply:
Internally, shader's texture is sampled, otherwise there will be aliasing

Reply:
"if Color == 0 && Brightness == 0 "  It is no use actually doing so